configs/rfnext/README.md

```diff
@@ -0,0 +1,177 @@
+            init_alphas=0.01,
+            mmin=1,
+            mmax=24,
+            num_branches=2,
+            skip_layer=[])),
+    )
+
+custom_hooks = [
+    dict(
+        type='RFSearch',
+        config=model['rfsearch_cfg']['config'],
+        mode=model['rfsearch_cfg']['mode'],
+        verbose=model['rfsearch_cfg']['verbose'],
+        by_epoch=model['rfsearch_cfg']['by_epoch']
+    ),
+]
+```
+
+Arguments:
+
+- `max_step`: The maximum number of steps to update the structures.
+- `search_interval`: The interval (epoch) between two updates.
+- `exp_rate`:  The controller of the sparsity of search space. For a conv with an initial dilation rate of `D`, dilation rates will be sampled with an interval of `exp_rate * D`.
+- `num_branches`: The controller of the size of search space (the number of branches). If you set `S=3`, the dilations are `[D - exp_rate * D, D, D + exp_rate * D]` for three branches. If you set `num_branches=2`, the dilations are `[D - exp_rate * D, D + exp_rate * D]`. With `num_branches=2`, you can achieve similar performance with less MEMORY and FLOPS.
+- `skip_layer`: The modules in skip_layer will be ignored during the receptive field search.
+
+## Training
+
+### Searching Jobs
+
+You can launch searching jobs by using config files with prefix `rfnext_search`. The json files of searched structures will be saved to `work_dir`.
+
+If you want to further search receptive fields upon a searched structure, please set `rfsearch_cfg.rfstructure_file` in config file to the corresponding json file.
+
+```shell
+bash ./tools/dist_train.sh \
+    ${rfnext_search_CONFIG_FILE} \
+    ${GPU_NUM}
+```
+
+### Training Jobs
+
+Setting `rfsearch_cfg.rfstructure_file` to the searched structure file (.json) and setting `rfsearch_cfg.mode` to `fixed_single_branch` or `fixed_multi_branch`, you can retrain a model with the searched structure.
+You can launch fixed_single_branch/fixed_multi_branch training jobs by using config files with prefix `rfnext_fixed_single_branch` or `rfnext_fixed_multi_branch`.
+
+```shell
+bash ./tools/dist_train.sh \
+    ${rfnext_fixed_single_branch_CONFIG_FILE} \
+    ${GPU_NUM}
+
+bash ./tools/dist_train.sh \
+    ${rfnext_fixed_multi_branch_CONFIG_FILE} \
+    ${GPU_NUM}
+```
+
+Note that the models after the searching stage is ready a `fixed_multi_branch` version, which achieves better performance than `fixed_single_branch`, without any retraining.
+
+**Warning: Please do not modify the values in `model.rfsearch_cfg` by specifying `--cfg-options`, because the values with the same keys in `custom_hooks` will not be modified by such operation.**
+
+## Inference
+
+`rfsearch_cfg.rfstructure_file` and `rfsearch_cfg.mode` should be set for inferencing stage.
+
+Single branch inference:
+
+```shell
+./tools/dist_test.sh \
+    ${rfnext_fixed_single_branch_CONFIG_FILE} \
+    ${CHECKPOINT_FILE} \
+    ${GPU_NUM} \
+    --out results.pkl \
+    --eval bbox segm \
+    --options "classwise=True"
+
```

Comment:
Seems the training and inference command are the same with other mmdetection models. We should emphasize the difference in config.